__tests__/lint/no-invalid-media-type-examples-invalid-schema/snapshot.js

```diff
@@ -3,7 +3,21 @@
 exports[`E2E lint no-invalid-media-type-examples-invalid-schema 1`] = `
 
 validating /openapi.yaml...
-[1] openapi.yaml:28:17 at #/paths/~1test/get/responses/202/content/application~1json/schema
+[1] openapi.yaml:16:17 at #/components/schemas/Test/nullable
+
+\`Type\` keyword should be explicitly defined when \`nullable\` keyword is used.
```

Comment:
```suggestion
The \`type\` field must be defined when the \`nullable\` field is used.
```
Change to this error message everywhere.